src/vorta/utils.py

```diff
@@ -344,7 +344,7 @@ def display_password_backend(encryption):
     # flake8: noqa E501
     if encryption != 'none':
         keyring = VortaKeyring.get_keyring()
-        return trans_late('utils', "Storing the password in your password manager.") if keyring.is_primary else trans_late(
+        return trans_late('utils', "Storing the password in your password manager.") if keyring.is_system else trans_late(
```

Comment:
'''
    keyring = VortaKeyring.get_keyring()
    if encryption != 'none' and not keyring.is_system:
        return trans_late('utils', "Storing the password in your password manager.") if keyring.is_primary else trans_late(
            'utils', 'Saving the password to disk. To store password more securely install a supported secret store such as KeepassXC')
    else:
        return ""
'''
Otherwise it always shows as using password manager when system keychain disabled.

Reply:
True. Will change.

Reply:
Looks convoluted. Moved this to the view where it's actually used. We have too many circular imports already.

Reply:
Don't move the backend, I'm reusing it in key management. Utils prevents circular imports.

Reply:
Sorry, but I didn't see any other uses of `display_password_backend()`. Which file do you mean?

Reply:
It's a pull request.

Reply:
I see. Which one? Duplicating a few lines isn't an issue.

Next task I want to do is to make imports more hierachical. Many imports are only possible within functions because everything imports from everywhere. So e.g. I don't want models or utils to import too much. They should be at the bottom.

I may have fixed the "hanger" during macOS testing as well. According to the debugger it was stuck waiting for the BorgThread mutex. So I tried a few things there, like using the native Python mutex, which has a nicer API.

Reply:
Never mind. Found `display_password_backend()` in your PR. Will try to put that function into the keyring class instead.

Reply:
Moved to `VortaKeyring().get_backend_warning()`